cmake-ide.el

```diff
@@ -665,21 +665,13 @@ the object file's name just above."
   "Get dir form hash table, if not present compute a build dir and insert it in the table.  For non cmake project, insert and use a nil entry (associated temp directory)."
   (let ((project-key (cmake-ide--get-project-key)))
     (when project-key
-      (let ((build-dir (gethash project-key cmake-ide--cmake-hash nil)))
```

Comment:
Without a test for this, there's no way of knowing if this is a breaking change or not. It's also not related to the PR's focus.

Reply:
> Without a test for this, there's no way of knowing if this is a breaking change or not. It's also not related to the PR's focus.

I see. I removed the cleanup commit.